dev-client/src/screens/ProjectInputScreen/SoilPitSettings.tsx

```diff
@@ -106,7 +106,8 @@ export const SoilPitSettings = ({
             width={'60%'}
             variant={'underlined'}
             selectedValue={settings.depthIntervalPreset}
-            onValueChange={onSelectUpdated(onOpen)}>
+            onValueChange={onSelectUpdated(onOpen)}
+            isDisabled={!userCanUpdateIntervals}>
```

Comment:
question: can we used enabled instead of disabled as a parameter? Should we used disabled instead of isdisabled to match HTML?